src/components/Dashboard/DashboardComponent.jsx

```diff
@@ -45,6 +43,10 @@ class DashboardComponent extends Component {
   ];
 
   render() {
+    const {
+      allocatedAssets, availableAssets, damagedAssets, lostAssets
```

Comment:
If you want to use multiline, then place each "destructured" value in its own line. I.e.
```js
const {
  allocatedAssets,
  availableAssets,
  damagedAssets,
  lostAssets
} = this.props;
```